_returners/splunk_pulsar_return.py

```diff
@@ -1,19 +1,27 @@
+# -*- encoding: utf-8 -*-
 '''
-
-Send json response data to Splunk via the HTTP Event Collector
-Requires the following config values to be specified in config or pillar:
-
-hubblestack:
-  pulsar:
-    returner:
-      splunk:
-        token: <splunk_http_forwarder_token>
-        indexer: <hostname/IP of Splunk indexer>
-        sourcetype: <Destination sourcetype for data>
-        index: <Destination index for data>
-
-Run a test by using salt-call test.ping --return splunk
-
+HubbleStack Pulsar-to-Splunk returner
+
+:maintainer: HubbleStack
+:maturity: 2016.7.0
+:platform: All
+:requires: SaltStack
+
+Deliver HubbleStack Pulsar event data into Splunk using the HTTP
+event collector. Required config/pillar settings:
+
+.. code-block:: yaml
+
+    hubblestack:
+      pulsar:
+        returner:
+          splunk:
+            token: <splunk_http_forwarder_token>
+            indexer: <hostname/IP of Splunk indexer>
+            sourcetype: <Destination sourcetype for data>
+            index: <Destination index for data>
+        return: splunk_pulsar
+        batch: True
```

Comment:
These lines don't belong here, do they? They need to go in the beacon config, which is not under the hubblestack heading.